### ISSUE TITLE
[CloudSql] Wrong field type for the books table

### ISSUE THREAD

**Issue by beastafk** [GoogleCloudPlatform/getting-started-php#157]
Hi, it seems like when testing the `2-structured-data` app, there is a small bug, which prevents the user from creating a Book record. The problem is that the `create()` method formats the `publishedDate` in the following format:

//controllers.php:66
```
$book['publishedDate'] = $d->setTimezone(
        	new \DateTimeZone('UTC'))->format("Y-m-d\TH:i:s\Z");
```

`string(20) "2017-05-02T21:00:00Z"`

And the insertion results in an error, because the `publishedDate` schema expects the format to be suitable for `DATE`:
```
$columns = array(
            'id INT NOT NULL AUTO_INCREMENT PRIMARY KEY ',
            'title VARCHAR(255)',
            'author VARCHAR(255)',
            'publishedDate DATE',
            'imageUrl VARCHAR(255)',
            'description VARCHAR(255)',
            'createdBy VARCHAR(255)',
            'createdById VARCHAR(255)',
        );
```

The `edit()` method has the same problem.

**Comment by bshaffer** [comment]
Hello! Thanks for reporting this. What data backend are you using?

**Comment by beastafk** [comment]
I was using MySQL, but it seems like you have fixed the issue in [f56fc6f19afab05d119afbf19d1ff5922f1be9ed](https://github.com/GoogleCloudPlatform/getting-started-php/commit/f56fc6f19afab05d119afbf19d1ff5922f1be9ed), so this can be closed now.
